install.md

```diff
@@ -51,6 +55,101 @@ If there's a version or operating system that is missing, please [open an issue]
 
 For more information, please follow the instructions in the [CRI-O packaging repository.](https://github.com/cri-o/packaging/blob/main/README.md).
 
+## Install CRI-O on Flatcar with Sysexts
+
+Installing CRI-O on Flatcar Linux with support for systemd extensions (sysexts),
+enabling a supported installation method for environments that utilize Flatcar.
+
+### Prerequisites
+
+- Flatcar Linux installed on your machine. [(Flatcare Installation guide)](https://www.flatcar.org/docs/latest/installing/)
+- Systemd extensions (sysexts) enabled on Flatcar Linux. [(Enable Sysext in Flatcar)](https://www.flatcar.org/docs/latest/provisioning/sysext/)
+- `Curl` or `wget` for downloading files.
+- `Tar` for extracting the CRI-O binaries.
+- `sed` for editing files in-place.
+- sufficient privileges (using sudo if necessary).
+
+Please make sure you have Flatcar Linux installed and sysexts enabled before
+proceeding with the installation of CRI-O.
+
+### Installation Steps
+
+To install CRI-O on Flatcar Linux with sysexts, follow these steps:
+
+- Step 1: Download the script
+Sample Extension script of install cri-o using sysext in [here](https://github.com/flatcar/sysext-bakery/blob/main/create_crio_sysext.sh)
+
+  - Using Curl:
+
+  ```bash
+  curl -L https://github.com/flatcar/sysext-bakery/blob/main/create_crio_sysext.sh
+  -o create_crio_sysext.sh
```

Comment:
Why?

```suggestion
  curl -L https://github.com/flatcar/sysext-bakery/blob/main/create_crio_sysext.sh -o create_crio_sysext.sh
```

Reply:
MD013/line-length error, line length should in under 80 char if i do it in suggested way it will be 85

Reply:
Right. This is silly then. Especially since there are things that can be easily wrapped in markdown.

Reply:
Sorry. But i couldn't get what you are suggesting up there in the last reply

```bash
curl -L \
https://github.com/flatcar/sysext-bakery/blob/main/create_crio_sysext.sh \
-o create_crio_sysext.sh
```

it is better this way ???

Reply:
Looks better. And in terms of my reply. I was simply being grumpy about silly linter rules.